### PR TITLE
DLC-994 replace colorbox with bootstrap modals

### PR DESCRIPTION
**Pull request by barmintor**
There's some API encapsulation to address, among other things, but for now just roughing out the use of Bootstrap modals instead of ColorBox.

- [x] Homepage projects "More Information" (replacing app/javascript/src/dcv/modals/project.js)
- [x] Item citations (replacing app/javascript/src/dcv/modals/citation.js)
- [x] Feedback & Suggestions (replacing app/javascript/src/dcv/modals/feedback.js)
- [x] MODS display (replacing app/javascript/src/dcv/modals/mods.js)
- [x] Zooming Image (replacing app/javascript/src/dcv/modals/zoomingImage.js)
- [x] Thumbnail Preview (replacing app/javascript/src/dcv/modals/preview.js)
- [x] ChildViewer (ColorBox invoked in app/javascript/src/dcv/childViewer.js)
- [x] FileSystem (ColorBox invoked in  app/javascript/src/ifp/dcv.filesystem.js)

### CODE REVIEW

**Comment by barmintor** [comment]
I think the homepage "More Information" one is basically done?

**Comment by barmintor** [comment]
I also think citations basically done. MODS is where you get into the issues with resizing around content, which will be a challenge for the image / childViewer situation. Feedback is in one sense basically done, but the Bootstrap modal wants to have a title. Would be interested in opinions about best way to approach that - could do tricks with margins, could add a title suppression param to feedback... IDK. Lots of options.

File: app/views/catalog/_featured_items.html.erb [hunk 1]
```diff
@@ -14,7 +14,7 @@
 								<%= content_tag(:span, "Browse #{digital_project[:name]} content", class: "sr-only", id: "#{digital_project_id_base}-search") -%>
 							<%- end -%>
 							<div class=" proj-jav btn-group d-flex" role="group">
-								<button class="btn btn-outline-secondary btn-sm" type="button" data-toggle="tooltip" aria-label="More Information" title="More Information" data-proj-more="#proj-<%=proj_counter%>" data-proj-title="<%= digital_project[:name] %>" onclick="return projectModal(this);"><span class="fa fa-info-circle" aria-hidden="true"></span></button>
+								<button class="btn btn-outline-secondary btn-sm" type="button" data-toggle="modal" aria-label="More Information" title="More Information" data-modal-body-delegate="#proj-<%=proj_counter%>" data-modal-title-delegate="#proj-<%= proj_counter %>-title" data-target="#dcvModal"><span class="fa fa-info-circle" aria-hidden="true"></span></button>
```
Comment (er-k) [inline app/views/catalog/_featured_items.html.erb:17]: tooltip removal on purpose?

Comment (barmintor) [reply app/views/catalog/_featured_items.html.erb:17]: Ack, no - but there does have to be a change (the button has data-toggle=modal, but the span needs data-toggle=tooltip)

Comment (barmintor) [reply app/views/catalog/_featured_items.html.erb:17]: That's what I did with the MODS XML widget in the item page, just need to do consistently

**Comment by barmintor** [comment]
FileSystem colorbox was ported over from SCV/cul_hydra and never used in production.

**Review by er-k — APPROVED** [review]

**Pull request merged**